### DISCUSSION TITLE
Search habbits by name

### DISCUSSION THREAD

**Discussion by dtgoitia** [iSoron/uhabits#758] [Feature Requests] [answered]
### Problem

I track quite a lot of (~50) habbits. Sometimes it takes me 10-15 secs to find the habbit I want to find.

### Solution

This search time could be dramatically improved with a simple-minimal search function. In my case, I know what the name of the habbit I am looking for, so it should take me no more than 2-3 secs to type and find the habbit.

### Feature request

Could a search function be added to quickly find/navigate to habbits?

**Comment by iSoron** [comment, marked as answer]
Duplicate of #676
